cibyl/sources/elasticsearch/api.py

```diff
@@ -164,6 +166,50 @@ def get_last_build(self: object, builds_jobs: AttributeDictValue):
 
         return AttributeDictValue("jobs", attr_type=Job, value=job_object)
 
+    def get_deployment(self, **kwargs):
+        """Get deployment information for jobs from elasticsearch server.
+
+        :returns: container of jobs with deployment information from
+        elasticsearch server
+        :rtype: :class:`AttributeDictValue`
+        """
+        jobs_to_search = []
+        if 'job_name' in kwargs:
+            jobs_to_search = kwargs.get('job_name').value
+        elif 'jobs' in kwargs:
+            jobs_to_search = kwargs.get('jobs').value
+
+        query_body = QueryTemplate('jobName', jobs_to_search).get
+
+        hits = self.__query_get_hits(
+            query=query_body
+        )
+
+        job_objects = {}
+        for hit in hits:
+            job_name = hit['_source']['jobName']
+            url = hit['_source']['envVars']['JOB_URL']
+            # If the key exists assign the value otherwise assign unknown
+            topology = hit.get('_source', {}).get('envVars', {}).get(
```

Comment:
Do all/most jobs have this variable defined in elasticsearch? In Jenkins I found that some have a `TOPOLOGY` variable.

Reply:
Yes, in this case the same data is located in both `envVars` and `buildParams` just with different variable name.

Reply:
Great, then elasticsearch should be more reliable than jenkins for deployment information